composeApp/src/main/kotlin/io/github/couchtracker/ui/screens/watchedItem/WatchedItemDialog.kt

```diff
@@ -2,117 +2,342 @@
 
 package io.github.couchtracker.ui.screens.watchedItem
```

Comment:
This file is called `WatchedItemDialog`, yet there's no composable called like that.